GOVERNANCE.md

```diff
@@ -0,0 +1,45 @@
+# Node.js Web Team Governance
+
+The Node.js Web Team (@nodejs/web) is a team in the Node.js Project that is composed by a set of subteams. Each containing specific responsibilities and goals.
+
+### Node.js Website Team (`@nodejs/nodejs-website`)
+
+The Node.js Website Team is responsible for the technical development of the Node.js Website. This includes the development of the features of the Website, providing stability and security to the Website, and the maintenance of the Website.
+
+The maintainers on the Node.js Website Team are responsible for steering the technical direction of the Node.js Website, and reserve the right to make final decisions on any issues or pull requests, in line with the Contribution Guidelines, Collaborator Guidelines, the Code of Conduct and the overall Governance premises of the Node.js project.
+
+Members of this team are nominated through the guidelines provided on the Collaborator Guidelines within this repository.
+
+### Node.js Web Infra Team (`@nodejs/web-infra`)
+
+The Node.js Web Infra Team is responsible for maintaining the Infrastructure revolving Node.js's Web Presence. (This includes the Node.js Website, for example). The Node.js Web Infra team has the responsibilites of:
+
+- Maintaining CI/CD pipelines related to Web Infrastructure
+- Maintaining our Infrastructure Providers (such as Vedrcel)
```

Comment:
It'd be nice to include Cloudflare here, but might be something to sync w/ Build WG on as to shared responsibility, as it also impacts a lot more than just the website.

Reply:
Feel free to add a a patch

Reply:
bump! @MattIPv4

Reply:
Ah, sorry!